Angular Electron not building on 2.1.5 Unexpected token: punc ()

On 2.1.5 my electron angular 7  app runs when npm start, but on production mode it fails like this:
`ERROR in main.cd7adcc1ce764ef292fd.js from Terser
Unexpected token: punc ()) [./node_modules/ytdl-core/lib/info.js:147,0][main.cd7adcc1ce764ef292fd.js:96082,4]
`
My typescript target is es2015

I think the problem is I can't update the target from es5, because Material Design 7 Tables won´t work. So , 3.0.1 fail because Omit is not present in this TS version .... 
I think it would be great to fix the 2 version as a ES5 compatible one.

node_modules/ytdl-core/typings/index.d.ts(163,70): error TS2304: Cannot find name 'Omit'.

--- a/lib/info.js
+++ b/lib/info.js
@@ -143,8 +143,7 @@ const gotConfig = async(id, options, info, body) => {
 
     // Get the day the vid was published.
     published: Date.parse(
-      info.player_response.microformat.playerMicroformatRenderer.publishDate,
-    ),
+      info.player_response.microformat.playerMicroformatRenderer.publishDate),
 
     // Get description.
     description: videoDetails.shortDescription,

